### ISSUE TITLE
Mesh Central on 3 monitors very difficult

### ISSUE THREAD

**Issue by CCWTech** [Ylianst/MeshCentral#1105]
Using Mesh central on a remote computer with 3 monitors is very difficult even though there are 3 monitors on the local side. With Remote Desktop you can tell it to use all monitors and it works very well. Is this a feature coming to Mesh Central?

**Comment by olivier37520** [comment]
As far as I know, this feature can’t be done. Meshcentral Remote Desktop runs into web browser. For that you must found a web browser supporting fullscreen across multiple display. Or use the nvidia desktop software.

**Comment by CCWTech** [comment]
Ok, thanks.



Please do not send a new request for help to this email, the response may be delayed. 

Need support? Please email mailto:support@ccwtech.com for a faster response.


Need a simple, effective & advanced Veterinary Clinical Management Platform? www.vetastic.com
Paying too much for phones? We now offer VOIP phone services.



Allen Crist, CEO

CCW Technology I.T. Services
Veterinary I.T. Specialists
Serving Small and Medium Business  since 2008

(801)410-0203 x5002


http://www.ccwtech.com

http://www.facebook.com/CCWTechnology




---- On Thu, 02 Apr 2020 15:31:35 -0600 olivier37520 <notifications@github.com> wrote ----




As far as I know, this feature can’t be done. Meshcentral Remote Desktop runs into web browser. For that you must found a web browser supporting fullscreen across multiple display. Or use the nvidia desktop software.

**Comment by krayon007** [comment]
This is something we are looking into for a future feature, but it requires significant retooling on the server, as it will basically need to support clipping and multiplexing server side.

**Comment by CCWTech** [comment]
Awesome, thanks!



Please do not send a new request for help to this email, the response may be delayed. 

Need support? Please email mailto:support@ccwtech.com for a faster response.


Need a simple, effective & advanced Veterinary Clinical Management Platform? www.vetastic.com
Paying too much for phones? We now offer VOIP phone services.



Allen Crist, CEO

CCW Technology I.T. Services
Veterinary I.T. Specialists
Serving Small and Medium Business  since 2008

(801)410-0203 x5002


http://www.ccwtech.com

http://www.facebook.com/CCWTechnology




---- On Thu, 02 Apr 2020 15:48:44 -0600 Bryan Roe <notifications@github.com> wrote ----




This is something we are looking into for a future feature, but it requires significant retooling on the server, as it will basically need to support clipping and multiplexing server side.

**Comment by Ylianst** [comment]
Right, the idea is to support having 3 browser's open each viewing a difference screen, but it's significant work. For now, the best is to try to stretch the one browser across all 3 monitors which is not ideal.

**Comment by CCWTech** [comment]
Will do, thanks for all you do!

**Comment by datapharmer** [comment]
I Would love to see this capability too. Perhaps multiple remote windows could be launched (one for each screen on the client) and share a session?

**Comment by robclay** [comment]
Update 0.9.75 May allow this:

https://twitter.com/MeshCentral/status/1487197178078511107?cxt=HHwWhoCq5Yb9yqMpAAAA
